python/Ganga/Core/GangaRepository/SubJobXMLList.py

```diff
@@ -345,6 +345,7 @@ def __call__(self, index):
 
     def __getitem__(self, index):
         """Return a subjob based upon index"""
+        logger.debug("__getitem__")
```

Comment:
In that case a more explanatory message would be useful? "Subjob being loaded" for example?